test/setup/platforms/index.ts

```diff
@@ -16,6 +17,8 @@ interface IPlatformSetup {
   clean: () => Promise<void>;
   // set up host requirements specific to this platform
   setupTester: () => Promise<void>;
+  // namespace where snyk-monitor will be deployed
+  namespace: () => Promise<string>;
```

Comment:
I think this isn't the right place to put it - the file relates only to the _platform setup_ and here we are mixing deployment with it ("namespace where snyk-monitor will be deployed")!

Reply:
Hum... I put it there because the namespace will be chosen depending on the platform that we're using. Maybe the comment is really bad. It might be `The namespace where snyk-monitor will be deployed depends on the platform is running the test`.

Reply:
I just think the whole concept of this module is to deal with the platform - e.g. raising a cluster, destroying it, etc. It shouldn't have the notion of namespaces because that's a deployment concern. And for deploying we have a separate `deployers` module.  I think some of our abstractions are starting to mix together that's all! If there isn't an obvious other place to put this function (figuring out which namespace we should deploy) then it could stay but to me it feels wrong 😬